### DISCUSSION TITLE
[Refacto] Reprendre la structure de la documentation / du site ?

### DISCUSSION THREAD

**Discussion by Clemog** [publicodes/publicodes#478] [Ideas]
J'ai noté plusieurs choses dans la doc:

- Pourquoi a-t-on une section `API` ?
- On distingue "Principes de base" qui contient des mécanismes et "Listes des mécanismes" dans `API`. Perso, je ne saisis pas bien la différence, dans NGC, certains mécanismes seraient plutôt des principes de base et inversement. Est-ce qu'il ne faudrait pas avoir une section "attributs" (unités, titres, icônes ...) et une section mécanismes ?
- Certains titres sont des pages (outils), les autres non
- Toutes les méthodes et fonctions utiles ne semble pas documentées ? Les types non plus ?

Peut-être que c'est le moment de revoir un peu plus la structure du site avec une documentation "plus épurée", une section "guides" et une section avec des articles plus haut niveau comme "Standards de conception des modèles de calcul publics"

J'ai l'impression que certaines discussions précédentes se rapprochent de certains points mentionnés ici, on pourra également faire un peu de tri

**Comment by johangirod** [comment]
J'ai commencé d'esquisser un squelette pour un nouveau site de doc : 
https://pad.numerique.gouv.fr/s/hU0DkSivE#

**Comment by johangirod** [comment]
J'ai créé les issues dédiées avec la milestone [« website-v2 »](https://github.com/publicodes/publicodes/milestone/2)
